learn/framework/index.md

```diff
@@ -1,9 +1,23 @@
 ---
 title: Appchain Development Framework
 description: Learn about the appchain Development Framework, Rust Programming Language, Substrate and its features, including XCM and ready-to-use pallets for your Runtime.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: octicons-code-24
+template: index-page.html
 ---
 
+# Appchain Development Framework
```

Comment:
```suggestion
# Network Development Framework
```

Please check the rest of this page and switch appchain > network, including the title and description at the top of the page